deployment.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: binary-bot
+  labels:
+    app: binary-bot
+spec:
+  replicas: 1
```

Comment:
Would not expect to see 1 replica normally - is this correct?
Should also be repelled from each other for HA.

Reply:
Please post the queries on issue section.